patterns/1-initial/principles.md

```diff
@@ -0,0 +1,147 @@
+
+## Forces
+
+Teams have trouble communicating exactly what the important aspects of
+InnerSource are.
+
+People lacking open source experience fail to understand what it means to bring
+open source best practices into the organisation.
+
+On a daily basis teams trying to follow InnerSource best practices have a hard
+time deciding if what they are doing is inline with general InnerSource values.
+
+## Solution
+
+Those involved with the InnerSource initiative in the organisation need to
+understand two perspectives:
+
+### Why does the organisation want to adopt InnerSource?
+
+In the past InnerSource has proven to be successful to solve several issues
+commonly found in organisations. Instead of going for generalizations, try to
+exactly identify the solutions that match the challenges of your organisation -
+preferably with those affected by the change you want to see. Some challenges
+that others have addressed by following InnerSource best practices:
+
+* Reduce development silos caused by excessive ownership culture.
+* Increase innovation speed by reducing time spent solving similar issues by
+  fostering healthy code reuse.
+* Increase development speed by better cross team collaboration.
+* Solve project/ team dependencies beyond "wait it out" and "build workarounds",
+  thereby reducing engineering bottlenecks.
+* Increase quality.
+* Increase employee happiness.
+* To increase success of new hires.
+* To build actionable documentation.
+
+### Provide clear InnerSource principles
+
+Once teams understand which problems InnerSource will help them address the next
+step is to explain which basic values help address these challenges. Based on
+basic open source development principles the following guidelines have been
+proven successful:
+
+1. Code must be transparently hosted within the organisation.
+
+Source code, documentation, data relevant for project development must be
+available and easy to find for anyone in the organisation.
+
+2. Contributions over feature requests.
+
+All stakeholders of a project act as potential contributors and are being
+treated and supported as such. Contributions remain suggestions instead of
+requirements. Coordination before a contribution helps avoid wasted effort.
+Projects provide contribution guidelines to avoid friction.
+
+3. Mistakes are opportunities for learning
+
+With work visible across the entire organisation any mistake is visible to
+everyone. As a result a culture must be established in which mistakes are
+opportunities for learning instead of failure that should be avoided at all
+cost.
+
+4. Allow written advise to accrete in a persistent, searchable archive
+
+All project communication, in particular decisions taken and discussions leading
+up to those decisions need to be archived. It must be possible to reference
+communication via stable URLs. Previous communication needs to be stored in a
+way that can easily be searched.
+
+5. Written over verbal communication
```

Comment:
I wonder if point 4 and 5 should change order?
With that you would first establish that things are written down and then second that the should be archived and searchable.

Reply:
Likely. It makes sense to first stress working in writing and then archiving what was written down.